lib/gems/pending/util/mount/miq_generic_mount_session.rb

```diff
@@ -490,6 +490,10 @@ def settings_mount_point
   end
 
   def source_for_log
-    @input_writer ? "<STREAMED_FROM_CMD>" : @source_input.path
+    case
```

Comment:
I think this looks better than an `if` would, but clearly `rubocop` disagrees.  Up to the reviewer if they want me to change it, but not going to go out of my way to do so.